integration/local/metamask-init.mjs

```diff
@@ -0,0 +1,28 @@
+import puppeteer from "puppeteer";
+import dappeteer from "dappeteer";
+
+export const metamaskInit = async () => {
+  try {
+    const browser = await dappeteer.launch(puppeteer, {
+      args: ["--no-sandbox"],
+      executablePath: process.env.PUPPETEER_EXEC_PATH,
+      headless: false, // must be false, so can test with metamask extension
+      defaultViewport: null,
+      slowMo: 10, // to see what's gg on
```

Comment:
should this be an option? otherwise it will always run in slowmo even when no one's looking at it?